packages/react-charting/src/utilities/TestUtility.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
This is great.
Abstracting the technical details will help speed up writing test cases.
1 point, add comments before each function specifying what it is meant for.

Reply:
thanks, done